### PR TITLE
set up get_next_id

### PR DESCRIPTION
**Pull request by curled-bash**
gets around `lastrowid` issue

### CODE REVIEW

File: src/confess.py [hunk 1]
```diff
@@ -32,13 +33,20 @@ async def confess(
 
     assert embed is not None
 
+    # lastrowid is the same as sequential_id
+    # TODO verify no issues with race conditions or whatever
+
+    embed.set_footer(text=f'ID: {database.get_next_id()}')
```
Comment (emmajsadams) [inline src/confess.py:39]: Something I just want to propose, not committed to this idea: Why do we need the post numbers at all? Do people actually use them or care? They could always include a link to the message if they want to reference another confession?

**Review by emmajsadams — APPROVED** [review]
this seems good to me! one quick thought that came into my mind though

File: src/confess.py (hunk 1 above)
Comment (curled-bash) [reply src/confess.py:39]: A link to a message is 
1. a sort of hidden discord feature 
2. ugly to look at compared to a number

In any case, it's a features that I think at least 3 people have asked for.

Comment (emmajsadams) [reply src/confess.py:39]: yeah its just that I don't actually see anyone referencing posts by number in the current system. Maybe I am overlooking it? regardless seems like you have most of it done so why not ship it

Comment (emmajsadams) [reply src/confess.py:39]: ok wait I changed my mind on this. Not only as you mentioned is there some request for this, but I think there is value in embedding the id from out database in the message. Useful for detecting bugs and serves as a sort of backup of the data just incase.

**Pull request merged**